i18n/qgis_es.ts

```diff
@@ -9315,7 +9315,7 @@ Por favor, configúrela antes de ejecutar algoritmos de GRASS.</translation>
     <message>
         <location filename="../python/plugins/processing/processing-i18n.cpp" line="610"/>
         <source>i.eb.hsebal01.coords - Computes sensible heat flux iteration SEBAL 01. Inline coordinates</source>
-        <translation type="unfinished"></translation>
+        <translation>i.eb.hsebal01.coords - Prodesa iteración SEBAL 01 de flujo de calor sensible. Coordendas en línea.</translation>
```

Comment:
please fix s/Prodesa/Procesa/

Reply:
Done in https://github.com/qgis/QGIS/pull/5834